src/main/java/org/zalando/nakadi/controller/SubscriptionController.java

```diff
@@ -71,7 +74,7 @@ public SubscriptionController(final SubscriptionDbRepository subscriptionReposit
         }
 
         try {
-            return createSubscription(subscriptionBase, request);
+            return createSubscription(subscriptionBase, request, client);
```

Comment:
I don't think that authorisation should be done at the service layer. I see authorisation as another level of feature. The same way that authentication is done in a AOP way, I think that decoupling authorisation for other business process is a good design principle.

I would rather have it in the controller, just to avoid mixing it with the service layer, than having it in the service.